config/zones/GB-NIR.yaml

```diff
@@ -49,6 +49,10 @@ emissionFactors:
       - datetime: '2021-01-01'
```

Comment:
As Nortern Ireland (GB-NIR) and Ireland (IE) is connected to the same grid, we're now using the same emission factor for both zones.